abce/gui/make_graphs.py

```diff
@@ -44,7 +44,7 @@ def make_aggregate_graphs(data, filename, ignore_initial_rounds):
         if col == 'index':
             continue
         title = make_title(filename, col)
-        plot = figure(title=title, sizing_mode='stretch_both',
+        plot = figure(title=title, width=1200, height=600, sizing_mode='stretch_both',
```

Comment:
You could have set the 1200 and 600 as the default arguments, so requiring fewer code changes.

Reply:
```python
def abce_figure(title):
    return figure(title=title, width=1200, height=600, sizing_mode='stretch_both')
```